ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml

```diff
@@ -201,15 +201,23 @@ tests:
       ASSISTED_CONFIG: |
         NUM_MASTERS=1
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-olm-suite
+- as: e2e-metal-assisted-olm-cnv
   optional: true
   run_if_changed: ^(src/assisted_installer_controller/.*)$
   steps:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
-        TEST_FUNC=test_olm_operator
-      PACKET_PLAN: m2.xlarge.x86
+        OLM_OPERATORS=cnv
+    workflow: assisted-baremetal
+- as: e2e-metal-assisted-olm-ocs
+  optional: true
+  run_if_changed: .*operator.*
```

Comment:
it should be the same as above, no? ``^(src/assisted_installer_controller/.*)$``

Reply:
or maybe the above should be updated